src/PkgServer.jl

```diff
@@ -77,7 +77,9 @@ function start(;kwargs...)
     update_registries()
 
     @sync begin
+        global last_registry_update
         @spawn while true
+            last_registry_update = now()
```

Comment:
Instead of updating it almost every second, I believe it makes more sense to update this only when there's a new registry in.

Reply:
We don't know when a new registry is published though; that's why we check in a loop.  This counter is just to help me debug if the registry update loop gets stuck somehow.